options.html

```diff
@@ -136,6 +136,37 @@
 {
   box-shadow: 0px 0px 10px #e03030;
 }
+#userFilterLists {
+    margin-bottom: 8px;
+}
+
+#whitelistForm #newWhitelistDomain,
```

Comment:
Detail: IDs should be unique across the entire document so no need to specify the parent's ID as well.